src/apps/aave/constants.ts

```diff
@@ -18,7 +18,7 @@ export const AAVE_V3_ADDRESSES_BY_NETWORK_ID: Record<
   [NetworkId['celo-alfajores']]: undefined,
   [NetworkId['ethereum-mainnet']]: {
     poolAddressesProvider: '0x2f39d218133afab8f2b819b1066c7e434ad94e9e',
-    uiPoolDataProvider: '0x91c0ea31b49b69ea18607702c5d9ac360bf3de7d',
+    uiPoolDataProvider: '0x5c5228ac8bc1528482514af3e27e692495148717',
```

Comment:
For some reason, the address listed in https://github.com/bgd-labs/aave-address-book/blob/fbb590953db44d62a756d4639cb77ea58afb299c/src/ts/AaveV3Ethereum.ts#L84 fails when querying data.

But the address listed in https://docs.aave.com/developers/deployed-contracts/v3-mainnet/ethereum-mainnet works 🤷